src/utils/findGitmojiCommand.ts

```diff
@@ -1,19 +1,38 @@
-// @flow
-import COMMIT_MODES from '@constants/commit'
+import { CommitModes } from '@constants/commit'
 import FLAGS from '@constants/flags'
 
-const isSupportedCommand = (command: ?string, options: Object): boolean => {
-  return Object.keys(options).includes(command)
+type Options = {
+  [key: string]: object
+}
+
+type Cli = {
+  flags: Record<string, object>
+  input: string[]
+  showHelp(): void
+}
+
+const isSupportedCommand = (
+  command: string | undefined,
+  options: Options
+): boolean => {
+  return command !== undefined && Object.keys(options).includes(command)
+}
+
+enum CommandType {
+  Flag = 'flag',
+  Command = 'command'
+}
```

Comment:
I would not recommend using enums and instead rely on `union` types for this.

https://www.youtube.com/watch?v=jjMbPt_H3RQ

```suggestion
type CommandType = 'flag' | 'command'
```